### DISCUSSION TITLE
Temp and Humidity sensor not showing

### DISCUSSION THREAD

**Discussion by Kurtroosbeek2** [tuya/tuya-home-assistant#385] [Q&A]
I have a Temperature and Humidity sensor.

   "category": "wsdcg",
        "create_time": 1628616792,
        "id": "bfba4a5151b0a78fdaohfs",
        "model": "TT081",
        "name": "sensy7",
        "node_id": "60a423fffe3c7341",
        "online": true,
        "owner_id": "4345234",
        "product_id": "gqhxixyk",
        "product_name": "Temp and Hum",

The device is linked to my account. It shows up in the project.
But it is not showing up in the Home Assistant.

**Comment by iotinkognito** [comment]
Raise as issue rather than discussion,
